src/platform_impl/linux/x11/window.rs

```diff
@@ -1833,7 +1833,7 @@ impl UnownedWindow {
     pub fn set_override_redirect(&self, value: bool) {
         let mut swa = ChangeWindowAttributesAux::default();
         swa.override_redirect = Some(value as u32);
-        self.xconn.xcb_connection()
+        let _ = self.xconn.xcb_connection()
```

Comment:
I prefer an `error` log message to be emitted in this case.

Reply:
done